scripts/discourse-sync/main.py

```diff
@@ -1,26 +1,31 @@
 import os
 import sys
 import yaml
-from pydiscourse import DiscourseClient
+from client import Client
 from pydiscourse.exceptions import DiscourseClientError
 
 
+# Staff notice to be added to every autogenerated doc
+STAFF_NOTICE = ("This doc was automatically generated from Juju code. Please "
+"do not try to edit it - instead, report any problems by leaving a reply "
+"below, and the Juju team will try to fix them as soon as possible.")
```

Comment:
You could do this as a multiline string, e.g.

```suggestion
STAFF_NOTICE = """This doc was automatically generated from Juju code. Please
do not try to edit it - instead, report any problems by leaving a reply
below, and the Juju team will try to fix them as soon as possible.
"""
```

It's probably a little nicer. It does mean that it'll contain newlines - I'm not sure what Discourse will do with those. If they make display worse then it's probably better keeping it as you have it than processing it to remove them.